### PR TITLE
primer: Hypothesis now requires Python>=3.8

### PR DESCRIPTION
**Pull request by ichard26**
### Description

looks like their project dev tooling uses some newer syntax or something

### Checklist - did you ...

- [x] Add a CHANGELOG entry if necessary? -> n/a
- [x] Add / update tests if necessary? does primer count?
- [x] Add new / update outdated documentation? -> n/a

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,13 +7,18 @@ _build
 docs/_static/pypi.svg
 .tox
 __pycache__
+
+# Packaging artifacts
 black.egg-info
+black.dist-info
```
Comment (ichard26) [inline .gitignore:13]: I've been spending a lot time recently in the Python packaging land hence this addition :)

**Review by JelleZijlstra — APPROVED** [review]

**Comment by ichard26** [comment]
Fuzz and tests are broken on Python 3.9 due to a change between CPython 3.9.7 and 3.9.8 removing a private API typed-ast depends on. Merging to unblock at least primer CI even if CI will still be red tonight :)

**Review by David8166 — COMMENTED** [review]
K

**Comment by cooperlees** [comment]
Maybe we could teach primer to parse out the classifiers in setup.* and respect them.

**Pull request merged**